### DISCUSSION TITLE
The installation manual on `next.shadcn-vue.com` doesn't actually work

### DISCUSSION THREAD

**Discussion by gyenabubakar** [unovue/shadcn-vue#952] [General]
I tried using the Sidebar components in the stable version but the styles weren't working as expected. For example, there's a `data-[state=open]:bg-sidebar-accent` class on a `SidebarMenuButton` that has no effect because on the `SidebarMenuButton`, the `data-state` is always `closed` regardless of the current state of the sidebar.

So, I decided to setup the `1.0.0-next.0` version instead, running all commands with `shadcn-vue@next`. First of all, if I add a `tsConfigPath` property and then try adding a new component, it fails with `invalid components.json file`. When I remove that property, the `components` and `lib` folders are created outside the project directory.

**Comment by sadeghbarati** [comment]
It should be fixed in the new CLI release see 

https://github.com/unovue/shadcn-vue/pull/917/commits/8a8d1ef20a0c42126f5ebff7233917b7a73ba86f
